packages/apt/test.sh

```diff
@@ -152,20 +152,23 @@ echo "::group::Upgrade"
 
 sudo apt purge -V -y ${pgroonga_package}
 
-# Disable upgrade test for first time packages.
-# TODO: Automate this. We don't want to maintain this manually.
-# case ${postgresql_version} in
-#   16) # TODO: Remove this after 3.1.4 release.
-#     exit
-#     ;;
-# esac
-
-sudo apt install -V -y ${pgroonga_package}
-createdb upgrade
-psql upgrade -c 'CREATE EXTENSION pgroonga'
-apt install -V -y \
-  ${repositories_dir}/${os}/pool/${code_name}/*/*/*/*_${architecture}.deb
-psql upgrade -c 'ALTER EXTENSION pgroonga UPDATE'
+if sudo apt install -V -y ${pgroonga_package}; then
```

Comment:
Could you tell me if there is the better way to check whether the package exists or not?
I tried to check the following commands but it didn't work.
- `apt install --dry-run package`: It succeeded using it
- `apt-cache show package`: It succeeded using it

Reply:
I think that at least this approach isn't suitable. Because we can't detect why it's failed when it's failed. It may be failed in normal package installation process instead of "the packages doesn't exist".

Reply:
fix: [8599cb2](https://github.com/pgroonga/pgroonga/pull/484/commits/8599cb2d69de82f3e394d6e1392f15402ddbee8b)
I've just fixed like that I check whether the package exists or not after purging the info. 
Because I didn't read we had already installed the package in the precious step and got the meta information about the package.